test/demo/service.yaml

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: demo-service
+  labels:
+    app: demo
+spec:
+  type: NodePort
+  ports:
+  - nodePort: 31000
```

Comment:
The system will select a random port for us, we don't need to specify the nodePort.